yaptide/admin/db_manage.py

```diff
@@ -28,7 +28,7 @@ class TableTypes(Enum):
     Page = auto()
 
 
-def connect_to_db(verbose: int = 0):
+def connect_to_db(verbose: int = 0) -> tuple(db.Connection, db.MetaData, db.Engine):
```

Comment:
I tried to add user using tool and got.


```
root@5ff3c5fb3ae7:/usr/local/app# python yaptide/admin/db_manage.py --help
Traceback (most recent call last):
  File "/usr/local/app/yaptide/admin/db_manage.py", line 31, in <module>
    def connect_to_db(verbose: int = 0) -> tuple(db.Connection, db.MetaData, db.Engine):
TypeError: tuple expected at most 1 argument, got 3
```

`tuple` is a class, therefore this code doesn't do what we want.
It should be:
```suggestion
def connect_to_db(verbose: int = 0) -> tuple[db.Connection, db.MetaData, db.Engine]:
```

I suggested `typing.Tuple` before, because it generated hints in IDE